tests/libtest.sh

```diff
@@ -100,13 +100,17 @@ assert_has_dir () {
 
 assert_not_has_file () {
     if test -f "$1"; then
-	echo 1>&2 "File '$1' exists"; exit 1
+        sed -e 's/^/# /' < "$1" >&2
+        echo 1>&2 "File '$1' exists"
```

Comment:
I did it this way round so that the reason for failure ($1 exists) is closest to the notification that the build has failed (presumably at the end). After a failed build on an autobuilder, you're probably reading the log in reverse (or at least that's what I do).

If you'd strongly prefer the echo before the sed I can do that, though.